### PR TITLE
Implement exists & existsSync.

### PR DESCRIPTION
**Pull request by hyunjunekim**
For #28.

### CODE REVIEW

File: src/js/fs.js [hunk 1]
```diff
@@ -59,6 +59,37 @@ fsBuiltin._createStat = function(stat) {
 };
 
 
+fs.exists = function(path, callback) {
+  if (!path || !path.length) {
+      process.nextTick(function () {
+                         if (callback) callback(false);
```
Comment (ILyoan) [inline src/js/fs.js]: Please fix indentation in accordance with other code.

File: src/js/fs.js [hunk 2]
```diff
@@ -59,6 +59,37 @@ fsBuiltin._createStat = function(stat) {
...
+                       });
+      return;
+  }
+
+  var cb = function(err, stat) {
+             if (callback) callback(err ? false : true);
```
Comment (ILyoan) [inline src/js/fs.js]: Here too

**Comment by wateret** [comment]
Squash commits into one, please.

**Comment by hyunjunekim** [comment]
@wateret Please Looks At This!. :) Thank you.

**Comment by wateret** [comment]
LGTM :)

File: test/run_pass/test_fs_exists.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+/* Copyright 2015 Samsung Electronics Co., Ltd.
```
Comment (LaszloLango) [inline test/run_pass/test_fs_exists.js]: Please update the copyright

File: test/run_pass/test_fs_existssync.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+/* Copyright 2015 Samsung Electronics Co., Ltd.
```
Comment (LaszloLango) [inline test/run_pass/test_fs_existssync.js]: ditto

**Comment by LaszloLango** [comment]
LGTM, please rebase to the current master and update the copyright, so we can land.

**Pull request closed without merging**